mne/utils/docs.py

```diff
@@ -3136,42 +3153,23 @@ def _reflow_param_docstring(docstring, has_first_line=True, width=75):
     ``average=True``.
 """
 
-_sphere_header = (
-    'sphere : float | array-like | instance of ConductorModel | None')
-_sphere_desc = (
-    'The sphere parameters to use for the head outline. Can be array-like of '
-    'shape (4,) to give the X/Y/Z origin and radius in meters, or a single '
-    'float to give just the radius (origin assumed 0, 0, 0). Can also be an '
-    'instance of a spherical :class:`~mne.bem.ConductorModel` to use the '
-    'origin and radius from that object.'
-)
-_sphere_topo = _reflow_param_docstring(
-    f"""{_sphere_desc} ``None`` (the default) is equivalent to
-    (0, 0, 0, {HEAD_SIZE_DEFAULT}).
-    Currently the head radius does not affect plotting.""",
-    has_first_line=False)
-_sphere_topo_auto = _reflow_param_docstring(
-    f"""{_sphere_desc} If ``'auto'`` the sphere is fit to digitization points.
-    If ``'eeglab'`` the head circle is defined by EEG electrodes ``'Fpz'``,
-    ``'Oz'``, ``'T7'``, and ``'T8'`` (if ``'Fpz'`` is not present, it will
-    be approximated from the coordinates of ``'Oz'``). ``None`` (the default)
-    is equivalent to ``'auto'`` when enough extra digitization points are
-    available, and (0, 0, 0, {HEAD_SIZE_DEFAULT}) otherwise. Currently the head
-    radius does not affect plotting.""", has_first_line=False)
-docdict['sphere_topomap'] = f"""
-{_sphere_header}
-    {_sphere_topo}
-
-    .. versionadded:: 0.20
-"""
-
 docdict['sphere_topomap_auto'] = f"""\
-{_sphere_header} | 'auto' | 'eeglab'
-    {_sphere_topo_auto}
+sphere : float | array-like | instance of ConductorModel | None  | 'auto' | 'eeglab'
+    The sphere parameters to use for the head outline. Can be array-like of
+    shape (4,) to give the X/Y/Z origin and radius in meters, or a single float
+    to give just the radius (origin assumed 0, 0, 0). Can also be an instance
+    of a spherical :class:`~mne.bem.ConductorModel` to use the origin and
+    radius from that object. If ``'auto'`` the sphere is fit to digitization
+    points. If ``'eeglab'`` the head circle is defined by EEG electrodes
+    ``'Fpz'``, ``'Oz'``, ``'T7'``, and ``'T8'`` (if ``'Fpz'`` is not present,
+    it will be approximated from the coordinates of ``'Oz'``). ``None`` (the
+    default) is equivalent to ``'auto'`` when enough extra digitization points
+    are available, and (0, 0, 0, {HEAD_SIZE_DEFAULT}) otherwise. Currently the
+    head radius does not affect plotting.
```

Comment:
Is the text about radius not affecting plotting true? I haven't used plot_topomap for sometime now (I work on iEEG data now), but there was a time when radius affected the plotting. Many of the eeglab .set files I used had "invisible" head outlines because of this, as the channel positions were in mm or cm, but were interpreted by mne as meters (and so the head outline was the size of one channel in the topo plot).

Reply:
I wonder that too, but haven't had a chance to test it yet.

Reply:
@mmagnuski update: head radius definitely does affect plotting. I'll remove this statement.

Reply:
Thanks for confirming this!